helm-c-yasnippet.el

```diff
@@ -303,7 +303,11 @@ space match anyword greedy"
     (candidate-transformer . (lambda (candidates)
                                (helm-yas-get-transformed-list helm-yas-cur-snippets-alist helm-yas-initial-input)))
     (action . (("Insert snippet" . (lambda (template)
-                                     (yas-expand-snippet template helm-yas-point-start helm-yas-point-end)
+                                     (let ((start helm-yas-point-start)
+                                           (end helm-yas-point-end))
```

Comment:
Since these variables are set by `helm-yas-get-cmp-context`, I wasn't sure if I _should_ have changed that. Please let me know if that's the case.